website/statistics.py

```diff
@@ -94,10 +168,10 @@ def render_live_stats(self, user, class_id):
         return render_template(
             "class-live-stats.html",
             class_info={"id": class_id, "students": students, "collapse": collapse,
-                        "show_c1": show_c1, "show_c2": show_c2, "show_c3": show_c3},
+                        "show_c1": show_c1, "show_c2": show_c2, "show_c3": show_c3,
```

Comment:
Maybe not for this PR, but I'd say `show_c{x}` should not be a part of `class_info`, but could be a separate parameter maybe? What do you think?